events/jsonschema/io.mintmetrics.mojito/mojito_exposure/jsonschema/1-0-0

```diff
@@ -9,14 +10,20 @@
   "type" : "object",
   "properties" : {
     "waveId" : {
-      "type" : "string"
+      "description" : "A canonical ID of the test used in cookies and in experiment in reports",
+      "type" : "string",
+      "maxLength" : 255
```

Comment:
I think these max lengths are sane. Best we use these wider fields so values aren't truncated or the events are rejected. I can't think of any case where users would need more than 255 chars for any of these.

Our wave JSON schema is a more complete and imposes tighter constraints [1]. But users are still free to publish tests with larger values.

Have you got any thoughts on sticking with 255 chars?

1: https://github.com/mint-metrics/mojito-js-delivery/blob/master/wave.schema.json

Reply:
I usually try to set loose Snowplow limits as it is quite some work to update a schema in terms of data modeling. Happy to update it though. I'm not sure if the id can be different to the 36 characters that are set in your wave.schema.json?